modules/ROOT/pages/afm-upload-monitor.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
```

Comment:
INSTRUCTIONS for edit:

Only the paragraph at line 6 is in need of your skills. The rest I've not changed, and I know not why GitHub reports a change at line 54.